nightly-style-policy.md

```diff
@@ -0,0 +1,26 @@
+Initial PR(s) implementing new syntax filed against rust-lang/rust should
+generally leave the rustfmt subtree untouched. In cases that that need to
+modify rustfmt (for example, to fix compiler errors that come from adding
+new variants to AST representation), they should modify rustfmt in such a
+way to keep existing formatting verbatim.
+
+Rustfmt is allowed to implement nightly-only formatting behavior without that
+syntax being specified by the style guide. Initial PR implementors are 
+encouraged but not required to open a PR against rustfmt suggesting an initial
+formatting behavior, or formatting may later be implemented as a PR by anyone,
+pending approval of the implementation from T-rustfmt. T-style should be
+notified to approve the interim style proposed by these PRs, but this decision
+is not binding and may be revisited until the feature is stabilized and the
+formatting is codified in the style guide. 
```

Comment:
I think this covers the last conversation we had at the meeting today in that the rules aren't finalized until `feature.stabilized && style_guide.contains(rules)`

After reflecting a bit more on this, I think I'm on the same page as everyone else. I could understanding wanting to get style guide PRs merged ahead of time for various reasons (e.g. ensuring it's done and done before the stable release with the stabilized feature, bors queue times, etc.), so I think that makes sense.

I just wouldn't want to see us getting into the habit of merging style guide rules for syntax that was still a long ways out from stabilization (i.e. i'd probably object to merging a style guide pr for syntax that was generally believed to be 6 months away from potential stabilization)